LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/ModuleConnection/ModuleConnectionHost.cs

```diff
@@ -204,6 +207,22 @@ private bool TryUpdateConfigurationWithDesiredProperties(TwinCollection desiredP
             return false;
         }
 
+        private void UpdateProcessingDelayWithDesiredProperties(TwinCollection desiredProperties)
+        {
+            if (desiredProperties is null)
+            {
+                return;
+            }
+
+            var reader = new TwinCollectionReader(desiredProperties, this.logger);
+            if (reader.TryRead<int>(Constants.ProcessingDelayKey, out var processingDelay))
+            {
+                this.logger.LogDebug($"Updating processing delay for LNS {this.networkServerConfiguration.GatewayID} " +
+                    $"to {processingDelay} from desired properties of the module twin");
+                this.networkServerConfiguration.ProcessingDelayInMilliseconds = processingDelay;
```

Comment:
shouldn't we add range checking here? If the configured delay in the twin is < 0 for example, I would expect to use the default. 

And didn't we discuss that we should use the default, if it is not specified? so they need to opt out? otherwise, they need to take action to get the higher scalability.

Reply:
or fallback to 0

Reply:
Makes sense, will add a check there. Regarding values <0 I think it's safer to use the default but don't have a strong opinion on it.

Reply:
Done in 6fad78bcfa16585a84159b122f35a6dae0a5e1ad, using default delay if provided value is <0.